.github/workflows/gobump.yml

```diff
@@ -4,19 +4,33 @@ name: "Updates Go dependencies via gobump"
 on:  # yamllint disable-line rule:truthy
   workflow_dispatch:
   schedule:
-    # Every Sunday at 15:00
-    - cron: "0 15 * * 0"
+    # Every Monday at 6:00 PM (images release at 5:00)
```

Comment:
This is maybe a silly question, but why is this changing from Sun->Mon ?

Reply:
Of course sorry, I noticed that we tag `images` on Monday 5.0 while dep update was executed in the worst possible day, that is the day before. This aligns it as I was trying to explain in the comment (unsuccessfully).